Add estimate parameter to update_virtual_pressure_source

arskama opened the pull request:
spec PR: https://github.com/w3c/compute-pressure/pull/305

This PR add own pressure estimate parameter to the update_virtual_pressure_source command as specified in https://w3c.github.io/compute-pressure/?experimental=1#update-virtual-pressure-source

arskama committed:
Add estimate parameter to update_virtual_pressure_source

spec PR: https://github.com/w3c/compute-pressure/pull/305

This PR add own pressure estimate parameter to the
update_virtual_pressure_source command as specified in
https://w3c.github.io/compute-pressure/?experimental=1#update-virtual-pressure-source

arskama commented:
@foolip ptal.

past commented:
I think it's just that people are busy and haven't gotten around to reviewing this yet. I will try to get to it before the end of the week if nobody beats me to it.

Have you looked at the infrastructure test failures?

arskama committed:
Fix wrong update between sensor and pressure source

There was a clear mistake in updating update_virtual_sensor
instead of update_virtual_pressure_source in testdriver-extra.js

arskama committed:
remove unnecessary default value

arskama commented:
@past I don't see any obvious correlation between the infrastructure test failures and my patch.
I m just wondering if it is because of the default value `estimate = -1.0`

arskama committed:
fix closing bracket

arskama commented:
@foolip @jgraham can you place have a look? Thanks

arskama committed:
Fix wrong indent

arskama commented:
> I think it's just that people are busy and haven't gotten around to reviewing this yet. I will try to get to it before the end of the week if nobody beats me to it.
> 
> Have you looked at the infrastructure test failures?

https://github.com/web-platform-tests/wpt/pull/51306 has exactly the same amount of tests and same failures.

past commented on tools/wptrunner/wptrunner/testdriver-extra.js:
```diff
@@ -468,8 +468,8 @@
 
-    window.test_driver_internal.update_virtual_pressure_source = function(source_type, sample, context=null) {
-        return create_context_action("update_virtual_pressure_source", context, {source_type, sample});
+    window.test_driver_internal.update_virtual_pressure_source = function(source_type, sample, estimate = -1.0, context=null) {
```
Is -1.0 the right default value here? I couldn't tell from the spec why that would be the case.

arskama replied:
Agree! I removed it and checked it still works when rolling out in chromium.

Thanks

past commented:
> > I think it's just that people are busy and haven't gotten around to reviewing this yet. I will try to get to it before the end of the week if nobody beats me to it.
> > Have you looked at the infrastructure test failures?
> 
> #51306 has exactly the same amount of tests and same failures.

I agree that the infrastructure test failures are unrelated to this PR.

arskama committed:
remove default value for estimate
